yb_migrate/src/srcdb/source.go

```diff
@@ -104,3 +130,108 @@ func (s *Source) ParseURI() {
 
 	}
 }
+
+func parseSSLString(source *Source) {
```

Comment:
parseSSLString() and createTLSConf() is for mysql only, can be moved to srcdb/mysql.go

Reply:
These functions are only accessing the `Source` type. Hence I kept them closer to the `Source` type. I will consider making them methods of the `Source` type. For now, I will leave them as they are here.